server.js

```diff
@@ -5,6 +5,11 @@ if (typeof userFunctionUri === "undefined" || userFunctionUri.trim() === "") {
     throw "FUNCTION_URI envvar not set or empty. Aborting.";
 }
 
+const debug = process.env.NODE_DEBUG;
```

Comment:
can we simplify this to something like:

```js
console.debug = util.debuglog('riff')
```

Reply:
The only I have with that is that [`util.debuglog` writes to stderr](https://nodejs.org/api/util.html#util_util_debuglog_section) while `console.debug` is usually a `console.log` alias and writes to stdout.